src/qamqpqueue.cpp

```diff
@@ -328,10 +329,11 @@ int QAmqpQueue::options() const
     return d->options;
 }
 
-void QAmqpQueue::declare(int options)
+void QAmqpQueue::declare(int options, const QAmqpTable arguments)
```

Comment:
this should be a const& passed in:
`void QAmqpQueue::declare(int options, const QAmqpTable &arguments)`